modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -260,37 +217,38 @@ supply the correct client certificate and header:
    https://[MASTER]:8443/oauth/token/request
```

Comment:
You've left `[MASTER]` here

Reply:
@stlaz Is the 8443 port relevant here? Or can I just update to `https://<namespace_route>/oauth/token/request`?

Reply:
Good catch, I did not notice 8443 being here, it's 443 now and that's the default for HTTPS so you can just update to what you suggested :wink: